src/gui/wizard/owncloudwizard.cpp

```diff
@@ -68,6 +70,41 @@ OwncloudWizard::OwncloudWizard(QWidget *parent)
     setPage(WizardCommon::Page_Result, _resultPage);
     setPage(WizardCommon::Page_WebView, _webViewPage);
 
+    // Add keyboard shortcut to allow the user to cancel wizard
+    auto *closeWindowAction = new QAction(this);
+    closeWindowAction->setShortcut(QKeySequence("Ctrl+W"));
+    connect(closeWindowAction, &QAction::triggered, this, &OwncloudWizard::reject);
+    addAction(closeWindowAction);
+
+    // Add keyboard shortcut to allow the user to quit
+    auto *quitAction = new QAction(this);
+    quitAction->setShortcut(QKeySequence("Ctrl+Q"));
+    connect(quitAction, &QAction::triggered, this, &QApplication::quit);
+    addAction(quitAction);
+
+    // Enable "Cancel" button
+    // (Will be re-disabled on second page of wizard)
+    // reject() is set to quit() if no accounts are set up
+    OwncloudWizard::setOption(QWizard::CancelButtonOnLeft);
+
+    if (AccountManager::instance()->accounts().isEmpty()) {
+        if (Utility::isMac()) {
```

Comment:
I'd stick to "Exit" only, this is what's used throughout the application for this action.

Reply:
ForegroundBackground (which I've removed from this pull request) actually reveals the native macOS "Quit" commands, but I agree that changing the text is incidental to the main issue here.